src/collect.js

```diff
@@ -925,8 +925,7 @@ window.MediathreadCollect = {
         this.displayAsset = function(asset,index) {
             var assetUrl = asset.sources[asset.primary_type];
             if (typeof assetUrl !== 'undefined') {
-                // make sure to strip out any url params
```

Comment:
The comment seems really explicit...I wonder if there were issues on some site with the url params? We should probably do some sanity checks once the next extension version is ready to go.